tests/utils/command.go

```diff
@@ -103,34 +79,33 @@ func RegisterNodeRun() {
 
 		// Assumes that startCmd will launch a node with HTTP Port at [utils.DefaultLocalNodeURI]
 		healthClient := health.NewClient(DefaultLocalNodeURI)
-		healthy, err := health.AwaitReady(ctx, healthClient, healthCheckTimeout, nil)
+		healthy, err := health.AwaitReady(ctx, healthClient, HealthCheckTimeout, nil)
 		gomega.Expect(err).Should(gomega.BeNil())
 		gomega.Expect(healthy).Should(gomega.BeTrue())
 		log.Info("AvalancheGo node is healthy")
-
-		blockchainIds := make(map[string]string)
-		files, err := filepath.Glob("./tests/precompile/genesis/*.json")
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-
-		for _, file := range files {
-			basename := filepath.Base(file)
-			index := basename[:len(basename)-5]
-			blockchainIds[index] = CreateNewSubnet(ctx, file)
-		}
-
-		blockchainIDsBytes, err := json.Marshal(blockchainIds)
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		return blockchainIDsBytes
-	}, func(data []byte) {
-		err := json.Unmarshal(data, &BlockchainIDs)
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	})
 
-	// SynchronizedAfterSuite() takes two functions, the first runs after each test suite is done and the second
-	// function is executed once when all the tests are done. This function is used
-	// to gracefully shutdown the AvalancheGo node.
-	var _ = ginkgo.SynchronizedAfterSuite(func() {}, func() {
+	ginkgo.AfterSuite(func() {
 		gomega.Expect(startCmd).ShouldNot(gomega.BeNil())
 		gomega.Expect(startCmd.Stop()).Should(gomega.BeNil())
+		// TODO add a new node to bootstrap off of the existing node and ensure it can bootstrap all subnets
+		// created during the test
 	})
 }
+
+// RunTestCMD runs a given test command with the given rpcURI
+// It also waits for the test ping to succeed before running the test command
+func RunTestCMD(testCMD *exec.Cmd, rpcURI string) {
```

Comment:
Rather than moving this here, could we put this code inside of `runHardhatTests`. It's only used there and this is setting a very specific environment variable coupled to the logic of that function, so it doesn't make much sense to me to export this function.

Reply:
moved